stack.go

```diff
@@ -0,0 +1,30 @@
+package slogx
+
+import (
+	"fmt"
+	"log/slog"
+	"runtime"
+	"strings"
+)
+
+const (
+	KeyStack = "stack"
+	trim     = 3
+)
+
+// Stack returns a stack trace formatted as panic output.
+// It excludes a call of slogx.Stack() itself.
+func Stack() slog.Attr {
+	buf := make([]byte, 2048)
+	runtime.Stack(buf, false)
```

Comment:
This is wrong: you ignore returned value, but it should be used to cut `buf` length. Look at stdlib examples in my previous comment how to do this in one line.